dali/operators/reader/loader/webdataset_loader.cc

```diff
@@ -34,8 +34,8 @@ namespace detail {
 namespace wds {
 
 inline MissingExtBehavior ParseMissingExtBehavior(std::string missing_component_behavior) {
-  std::transform(missing_component_behavior.begin(), missing_component_behavior.end(),
-                 missing_component_behavior.begin(), static_cast<int (*)(int)>(std::tolower));
+  for (auto &c : missing_component_behavior)
+      c = std::tolower(static_cast<unsigned char >(c));
```

Comment:
```suggestion
      c = std::tolower(static_cast<unsigned char>(c));
```

Reply:
```suggestion
      c = std::tolower(static_cast<unsigned char>(c));
```
I wonder why the linter didn't scream. But I guess we can fix in a followup.